Resources/public/js/views/ez-userprofileview.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-userprofileview', function (Y) {
+    'use strict';
+    /**
+     * Provides the User Profile View class
+     *
+     * @module ez-userprofileview
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The User Profile View
+     *
+     * @namespace eZ
+     * @class UserProfileView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.UserProfileView = Y.Base.create('userProfileView', Y.eZ.TemplateBasedView, [], {
+        events: {
+            '.ez-user-profile': {
+                'tap': '_toggleUserMenu'
+            }
+        },
+
+        /**
+        * Renders the user profile view
+        *
+        * @method render
+        * @return {eZ.UserProfileView} the view itself
+        */
+        render: function () {
+            this.get('container')
+                .setHTML(this.template({
+                    user: this.get('user').toJSON(),
+                    avatarAltText: this.get('avatarAltText'),
+                    defaultAvatarImage: this.get('defaultAvatarImage')
```

Comment:
actually, that's a bit more complicated than that. I've just realized you are still displaying the original image variation (which can be big while the image is super small).
So, you should a dedicated image variation definition in `image_variations.yml` and load this variation from the user's `avatar` attribute (if filled) in the navigation hub view service. This variation should then be passed to the navigation hub view which pass it to the user profile view.